### PR TITLE
WRP-9708: Update 'ri' description in CLI docs

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Update 'ri' description in CLI docs

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Add `basesize` description.
- Replace `less plugin` to `postcss-resolution-independence`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
- Add `externalStartup` and `forceCSSModules` that already exist in CLI docs.

### Links
[//]: # (Related issues, references)
WRP-9708

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

### CODE REVIEW

File: packages/agate/template/README.md [hunk 1]
```diff
@@ -75,12 +75,14 @@ The @enact/cli tool will check the project's `package.json` looking for an optio
...
-* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"agate"`.
-* `fontGenerator` _[string]_ - Filepath to a CommonJS fontGenerator module which will build locale-specific font CSS to inject into the HTML. By default, will use any preset for a specified theme or fallback to agate.
-* `ri` _[object]_ - Resolution independence options to be forwarded to the [LESS plugin](https://github.com/enactjs/less-plugin-resolution-independence). By default, will use any preset for a specified theme or fallback to agate.
-* `screenTypes` _[array|string]_ - Array of 1 or more screentype definitions to be used with prerender HTML initialization. Can alternatively reference a json filepath to read for screentype definitions.  By default, will use any preset for a specified theme or fallback to agate.
+* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"sandstone"`.
```
Comment (MikyungKim) [inline packages/agate/template/README.md:78]: Why did you change agate to sandstone???

File: packages/cordova/template/README.md [hunk 2]
```diff
@@ -78,12 +78,14 @@ The @enact/cli tool will check the project's `package.json` looking for an optio
...
-* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"moonstone"`.
-* `fontGenerator` _[string]_ - Filepath to a CommonJS fontGenerator module which will build locale-specific font CSS to inject into the HTML. By default, will use any preset for a specified theme or fallback to moonstone.
-* `ri` _[object]_ - Resolution independence options to be forwarded to the [LESS plugin](https://github.com/enactjs/less-plugin-resolution-independence). By default, will use any preset for a specified theme or fallback to moonstone.
-* `screenTypes` _[array|string]_ - Array of 1 or more screentype definitions to be used with prerender HTML initialization. Can alternatively reference a json filepath to read for screentype definitions.  By default, will use any preset for a specified theme or fallback to moonstone.
+* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"sandstone"`.
```
Comment (MikyungKim) [inline packages/cordova/template/README.md:81]: Was this template moonstone or not?

File: packages/moonstone/template/README.md [hunk 3]
```diff
@@ -75,12 +75,14 @@ The @enact/cli tool will check the project's `package.json` looking for an optio
...
-* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"moonstone"`.
-* `fontGenerator` _[string]_ - Filepath to a CommonJS fontGenerator module which will build locale-specific font CSS to inject into the HTML. By default, will use any preset for a specified theme or fallback to moonstone.
-* `ri` _[object]_ - Resolution independence options to be forwarded to the [LESS plugin](https://github.com/enactjs/less-plugin-resolution-independence). By default, will use any preset for a specified theme or fallback to moonstone.
-* `screenTypes` _[array|string]_ - Array of 1 or more screentype definitions to be used with prerender HTML initialization. Can alternatively reference a json filepath to read for screentype definitions.  By default, will use any preset for a specified theme or fallback to moonstone.
+* `theme` _[object]_ - A simplified string name to extrapolate `fontGenerator`, `ri`, and `screenTypes` preset values from. For example, `"sandstone"`.
```
Comment (MikyungKim) [inline packages/moonstone/template/README.md:78]: Please don't make any changes to the theme names.

**Review by MikyungKim — CHANGES_REQUESTED** [review]
Please, don't make any changes to the theme name. It's based on the template's theme.
Where is the description for `baseSize`?

**Comment by seunghoh** [comment]
Nice try, but I might want to rewrite the `ri` description not just modify this PR.

**Pull request closed without merging**